package/src/contexts/overlayContext/OverlayProvider.tsx

```diff
@@ -167,9 +167,6 @@ export const OverlayProvider = <
     }
   }, [overlay]);
 
-  // Setup translators
-  useStreami18n({ i18nInstance, setTranslators });
```

Comment:
This is redundant!!

Reply:
Pointed this out when I was reviewing. I thought there was a reason you did this change but yes now it's clear.